about.html

```diff
@@ -0,0 +1,162 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Material+Icons+Outlined">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@100;300&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="/mob.css">
+    <link rel="stylesheet" href="/about.css">
+    <link rel="stylesheet" media="(min-width:768px)" href="/aboutDesktop.css">
+    <link rel="stylesheet" media="(min-width:768px)" href="/des.css">
+    <title>About</title>
+</head>
+<body>
+    <header>
+        <div class="top">
+            <div class="media">
+                <ul class="mediaList">
+                    <li>
+                        <div class="mediaIcon">
+                            <a class="face" href="#"><img src="images/png/facebook.png" alt="Facebook icon"></a>
+                        </div>
+                    </li>
+                    <li>
+                        <div class="mediaIcon twit">
+                            <a href="#"><img src="images/png/twitter.png" alt="Twitter icon"></a>
+                        </div>
+                    </li>
+                    <li><a href="#">English</a></li>
+                    <li><a href="#">My Page</a></li>
+                    <li><a href="#">Logout</a></li>
+                </ul>
+            </div>
+            <div class="desktopBar">
+                <div class="logo">
+                    <a href="index.html"><img src="images/png/logo1.png" height="75px"  alt="sony logo icon"></a>
+                </div>
+                <ul class="deskOptions">
+                    <li>
+                        <a href="index.html" class="deskLi">Home</a>
+                    </li>
+                    <li>
+                        <a href="#" class="deskLi">Program</a>
+                    </li>
+                    <li>
+                        <a href="#" class="deskLi">Join</a>
+                    </li>
+                    <li>
+                        <a href="#" class="deskLi">Sponsors</a>
+                    </li>
+                    <li>
+                        <a href="#" class="deskLi">contact</a>
+                    </li>
+                </ul>
+                <a class="rock deskLi" href="index.html">Sony Music</a>
+            </div>
+            <div class="small">
+                <a class="home" href="index.html"><span class="material-icons-outlined">home</span></a>
```

Comment:
Great job @sonuengineer 🤝

- [ ] Your design tends to be having a hanging image on your about desktop view. Kindly fix this hanging image and your design will look better.
![Hanging-Image](https://user-images.githubusercontent.com/63917725/205430834-b5a93e47-f5a8-4420-91a6-495a541f1c78.PNG)